src/main/java/org/codehaus/groovy/runtime/DefaultGroovyMethods.java

```diff
@@ -2996,15 +2998,6 @@ public static Number count(Iterator self, Object value) {
         return answer;
     }
 
-    /**
-     * @deprecated use count(Iterable, Closure)
-     * @since 1.0
-     */
-    @Deprecated
-    public static Number count(Collection self, Object value) {
```

Comment:
We could remove the method in 4.0.0, but it's better to remain it in the released 3.0.x for binary compatibility